### ISSUE TITLE
GZip and Flate Need Required.

### ISSUE THREAD

**Issue by nolyoi** [kemalcr/kemal#583]
### Description
Kemal doesn't run "out of the box". Some extra requires are necessary that should be included when you require Kemal. Seems this line isn't working?
```
{% if compare_versions(Crystal::VERSION, "0.35.0-0") >= 0 %}
  require "compress/deflate"
  require "compress/gzip"
{% end %}
```

### Steps to Reproduce

1. Install kemal shard
2. Run

**Expected behavior:** To work out of the box.

**Actual behavior:** Need to require Flate and GZip.
```
In lib/kemal/src/kemal/helpers/helpers.cr:143:7

 143 | Gzip::Writer.open(env.response) do |deflate|
       ^
Error: undefined constant Gzip::Writer
```
```
In lib/kemal/src/kemal/helpers/helpers.cr:148:7

 148 | Flate::Writer.open(env.response) do |deflate|
       ^
Error: undefined constant Flate::Writer
```

### Versions
**Crystal:** 0.35.1
**Kemal:** 0.26.1
**OS:** Pop_OS 20.04 (Ubuntu 20.04)

**Comment by mamantoha** [comment]
Hi @nolyoi use kemal from the `master` branch.

Add this to your application's `shard.yml`:

```yaml
dependencies:
  kemal:
    github: kemalcr/kemal
    branch: master
```

**Comment by nolyoi** [comment]
> Hi, @nolyoi use Kemal from the `master` branch.
> 
> Add this to your application's `shard.yml`:
> 
> ```yaml
> dependencies:
>   kemal:
>     github: kemalcr/kemal
>     branch: master
> ```

Still not showing up in the helpers file with branch: master added. Just deleted kemal, reinstalled with branch: master and same issue.

**Comment by Blacksmoke16** [comment]
Duplicate of:
- #573
- #572
- #571

Fixed by #570, just waiting on a new release.  Can use `branch`, or `commit` options as mentioned in #572.

**Comment by Xosmond** [comment]
@Blacksmoke16 Its September, why didn't you release this?

**Comment by Blacksmoke16** [comment]
Well for one this isn't my project and I don't have permission to do so.  I think you meant to ask @sdogruyol.

**Comment by kingsleyh** [comment]
@sdogruyol will you be releasing this soon?

**Comment by vinyll** [comment]
This pretty much breaks the basic quick tutorial.
That's refraining newcomers, that would be great to be merged 🙏

**Comment by vinyll** [comment]
As mentioned in previous comments the following combination did the job for me too:

```
dependencies:
  kemal:
    github: kemalcr/kemal
    branch: master
```

Then reinstall with

```
shards update
```

**Comment by kingsleyh** [comment]
Maybe @sdogruyol is taking a break

**Comment by ricardobeat** [comment]
@sdogruyol I'd be happy to help get this out if you don't have the time.

**Comment by ricardobeat** [comment]
In the meantime, use this to also avoid an issue with websockets (#577):

```
dependencies:
  kemal:
    github: kemalcr/kemal
    commit: dfe7dca
```

**Comment by mamantoha** [comment]
@ricardobeat It's already fixed in master branch. You can update your project's `shard.yml`

```
dependencies:
  kemal:
    github: kemalcr/kemal
    branch: master
```

and run

`shards update`.

That's it.

@sdogruyol should make a new release with the latest changes.

**Comment by otherguy** [comment]
Would be nice. It's November.

**Comment by ricardobeat** [comment]
@bcardiff @sdogruyol is something going on, anything we can do to help? There have been no updates to the project since August.

**Comment by bcardiff** [comment]
I'm not the maintainer of the repo, nor have access to the organization. @sdogruyol is the one who can do releases.

**Comment by matteofuzz** [comment]
In my company we're evaluating more use of Crystal and Kemal, but this issue is concerning, is the project maintained? What's in the future?

**Comment by sdogruyol** [comment]
It's been a while, I'll maintain and triage the issues. Also adding more people as maintainers.

**Comment by sdogruyol** [comment]
Closing as the original issue is already resolved 👍
